### ISSUE TITLE
Changes required in profile section #SWoC20

### ISSUE THREAD

**Issue by preeti13456** [preeti13456/CityonBikes#35]

**Comment by Sloth-Panda** [comment]
would like to work on this

**Comment by dollyjain51099** [comment]
I would like to work on this issue, please assign it to me!

**Comment by preeti13456** [comment]
ok, Since @Sloth-Panda  has been already assigned to another issue, I will be assigning this issue to you @dollyjain51099 but you have to first share your approach.

**Comment by dollyjain51099** [comment]
Can you please give a brief about this issue.

**Comment by preeti13456** [comment]
Did you check the website completely?

**Comment by preeti13456** [comment]
Basically when we go to the profile section of the website, it's not work fine on mobile. As well as it's not quite appealing

**Comment by preeti13456** [comment]
So you have to work on that.

**Comment by dollyjain51099** [comment]
okay, thank you for the help!

**Comment by dollyjain51099** [comment]
> Basically when we go to the profile section of the website, it's not work fine on mobile. As well as it's not quite appealing

I'm not able to view any profile section, so can you please tell me whether I should create one!

**Comment by preeti13456** [comment]
Yes instead of creating one work on the source code and check why it's not working on mobile. Is still ot didn't work out you can proceed but don't create a a separate folder for the same.

**Comment by dollyjain51099** [comment]
But there is no specific profile page that exists.

**Comment by preeti13456** [comment]
Check the templates folder there will be some pages regarding home and about us

**Comment by dollyjain51099** [comment]
but how are you co-relating home.html and about.html with profile page?

**Comment by preeti13456** [comment]
Basically it's the about us page but i was saying about home page as you have raised the issue that it's not accessible through mobile.

**Comment by preeti13456** [comment]
The page about us was already modified you have to work on regex.html page.

**Comment by nehaparmar99** [comment]
Hey, can I work on this issue?

**Comment by preeti13456** [comment]
Is there any update @dollyjain51099 ?

**Comment by dollyjain51099** [comment]
Can I contact you on Slack? @preeti13456

**Comment by preeti13456** [comment]
Sure

Get Outlook for Android<https://aka.ms/ghei36>
________________________________

**Comment by dollyjain51099** [comment]
I contacted you on Slack. @preeti13456

**Comment by preeti13456** [comment]
Update?

**Comment by Abhir-24** [comment]
@preeti13456 if only the UI of the profile section needs to be changed then I can work on this issue using HTML, CSS

**Comment by Ajaykumar34** [comment]
can i work on this issue ?

**Comment by preeti13456** [comment]
Sure

**Comment by anirudhsai20** [comment]
Hi @preeti13456 . Can I also work on this issue under SWOC please?

**Comment by preeti13456** [comment]
Sure
